script/test.py

```diff
@@ -0,0 +1,26 @@
+__author__ = "Xinqiang Ding <xqding@umich.edu>"
+__date__ = "2018/12/17 22:29:44"
```

Comment:
Consider updating these metadata as appropriate to the authorship, etc. and also adding a file/top-level docstring to describe the purpose of this file.